src/acl/index.js

```diff
@@ -8,11 +8,18 @@ module.exports = (web3) => {
     return new web3.eth.Contract(require('@aragon/os/build/contracts/ACL').abi, aclAddr)
   }
 
+  const getRoleId = async (repoAddr) => {
+    const repo = new web3.eth.Contract(require('@aragon/os/build/contracts/AragonApp').abi, repoAddr)
+    const roleId = await repo.methods.CREATE_VERSION_ROLE().call()
+  }
+
   return {
-    grant: async (repoAddr, grantee) => {
+    grant: async (repoAddr, grantee, useLegacyRoleId = false) => {
       const acl = await getACL(repoAddr)
 
-      const roleId = '0x0000000000000000000000000000000000000000000000000000000000000001'
+      const roleId = useLegacyRoleId ?
+        '0x0000000000000000000000000000000000000000000000000000000000000001' :
+        await getRoleId(repoAddr)
```

Comment:
@izqui in #158 you've said:

> Or even better, a call can be made to the Repo contract to get the id for the role.

While this sounds more like an either/or scenario, I've now implemented both ideas: getting hash id from the contract and explicitly falling back to the legacy id using `--legacy-role`.

It wasn't entirely clear to me if both were desired or just one of those options. Let me know if you want this to be changed.

Reply:
If we get the role hash directly from the contract there is no need to have an option for supporting the legacy role hash, as the contract will always return the actual role hash for the repo. Sorry the issue description wasn't clear enough 😅

I think we should drop the legacy flag and always use the role of the contract.

Reply:
Makes sense. I'll remove it.